parm/config/gfs/config.resources

```diff
@@ -118,26 +116,35 @@ case ${step} in
 
   "waveprep")
     export wtime_waveprep="00:10:00"
-    export npe_waveprep=5
+    export npe_waveprep_gdas=5
     export npe_waveprep_gfs=65
-    export nth_waveprep=1
+    export nth_waveprep_gdas=1
     export nth_waveprep_gfs=1
-    export npe_node_waveprep=$(( npe_node_max / nth_waveprep ))
+    export npe_node_waveprep_gdas=$(( npe_node_max / nth_waveprep_gdas ))
     export npe_node_waveprep_gfs=$(( npe_node_max / nth_waveprep_gfs ))
-    export NTASKS=${npe_waveprep}
+    export NTASKS_gdas=${npe_waveprep_gdas}
     export NTASKS_gfs=${npe_waveprep_gfs}
-    export memory_waveprep="100GB"
+    export memory_waveprep_gdas="100GB"
     export memory_waveprep_gfs="150GB"
+
+    var_npe_node="npe_node_waveprep_${RUN}"; var_nth="nth_waveprep_${RUN}"
+    var_npe="npe_waveprep_${RUN}"; var_NTASKS="ntasks_${RUN}"
```

Comment:
Yeah, that's probably better practice and more readable.  Fixed.

Reply:
Since these aren't `export`'d, they don't need to be.  However, I could see a situation where one of the `config.resources.MACHINE` files would need to set its own value for these, so I'll go ahead and unset them.

Reply:
Fixed.